spec/unit/concerns/connection_spec.rb

```diff
@@ -75,7 +75,7 @@
 
       it "must always be used last before the Faraday Adapter" do
         client.middleware.handlers.reverse.index(Restforce::Middleware::Logger).
-          should eq 1
+          should eq 0
```

Comment:
I'm not sure about this spec. @timrogers Can you take a look?

Reply:
This suggests that the Faraday adapter is missing - which could be a problem!

Reply:
@timrogers Handles and Adapters are separated now. 

`https://github.com/lostisland/faraday/blob/master/UPGRADING.md#others-1`

Reply:
In the connection file the builder already separates by pointing to the adapter. 

https://github.com/restforce/restforce/blob/master/lib/restforce/concerns/connection.rb#L64

Do you think this Faraday modification is a problem?